src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,6 +1,5 @@
 package com.kodilla.ecommercee;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
```

Comment:
Ja nie używam import z gwiazdką. Nie jest to jakiś poważny błąd ale przy aktualizacji bibliotek może się okazać, że Twój kod się nie skompiluje jeśli nagle ktoś doda do pakietu A klasę o takiej samej nawie, której używasz z pakietu B... wyskakują wtedy kominukaty ze słowem "ambigous". Ja radzę zawsze robić import z normalną nazwą klasy bez *. Jak jesteś doświadczony to szybko znajdziesz przyczynę błędu kompilacji ale może to być kłopotliwe.